integrated_channels/moodle/client.py

```diff
@@ -36,8 +36,8 @@ def inner(self, *args, **kwargs):
             # This cannot be converted to JSON thus the above fails miserably.
             # The above can fail with different errors depending on the format of the returned page.
             # Moodle of course does not tell us what is wrong in any part of this HTML.
-            raise ClientError('Moodle API task "{method}" failed due to unknown error.'.format(
-                method=method.__name__), response.status_code
+            raise ClientError('Moodle API task "{method}" for enterprise_configuration "{enterprise_configuration_id}" failed due to unknown error with code "{code}".'.format(
+                method=method.__name__, code=response.status_code, enterprise_configuration_id=self.enterprise_configuration.id), response.status_code
```

Comment:
i wanted to share some information on the customer impacted via the enterprise config record being used